recipes/nds2-client-swig/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "nds2-client-swig" %}
+{% set version = "0.16.0" %}
+{% set sha256 = "db785868f519a07e3a1cf7216cf5610a984d96e3795862794de98215534aa463" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - make
+    - cmake >=3.6
+    - pkg-config
+    - swig >=3.0.7
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - nds2-client ={{ version }}
+    - python
+    - numpy {{ numpy }}
```

Comment:
I think you need to use `pin_compatible('numpy')`: https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
@scopatz, that doc suggests just using `numpy` in the `host` section, and only using `pin_compatible` in the `run` dependencies, 1f30c18 updates the recipe to use `pin_compatible` in both sections, which works, but the builds seem to be using numpy-1.15.0. I would have thought they should use numpy-1.9.3?